ovos_bus_client/apis/events.py

```
@@ -0,0 +1,229 @@
+        message = self._get_source_message()
+        msg = message.forward('mycroft.scheduler.get_event', data)
+        status = self.bus.wait_for_response(msg, reply_type=reply_name)
+
+        if status:
+            event_time = int(status.data[0][0])
+            current_time = int(time.time())
+            time_left_in_seconds = event_time - current_time
+            LOG.info(time_left_in_seconds)
+            return time_left_in_seconds
+        else:
+            raise Exception("Event Status Messagebus Timeout")
```

Ruff (0.14.3)

Create your own exception

(TRY002)